# References: Prophage Gene Modules and Terminase-Defined Lineages

## Primary Data Sources

- Arkin AP et al. (2018). "KBase: The United States Department of Energy Systems Biology Knowledgebase." *Nat Biotechnol* 36:566-569. PMID: 29979655
- Parks DH et al. (2022). "GTDB: an ongoing census of bacterial and archaeal diversity through a phylogenetically consistent, rank normalized and complete genome-based taxonomy." *Nucleic Acids Res* 50:D199-D207. PMID: 34520557

## Prophage Biology and Distribution

- Bobay L-M et al. (2014). "Pervasive domestication of defective prophages by bacteria." *PNAS* 111:12127-12132. PMID: 25092302
- Casjens SR (2003). "Prophages and bacterial genomics: what have we learned so far?" *Mol Microbiol* 49:277-300.
- Casjens SR, Gilcrease EB (2009). "Determining DNA packaging strategy by analysis of the termini of the chromosomes in tailed-bacteriophage virions." *Methods Mol Biol* 502:91-111.
- Casjens SR, Hendrix RW (2015). "Bacteriophage lambda: Early pioneer and still relevant." *Virology* 479-480:310-330.
- Howard-Varona C et al. (2017). "Lysogeny in nature: mechanisms, impact and ecology of temperate phages." *ISME J* 11:1511-1520.
- Kim MS, Bae JW (2018). "Lysogeny is prevalent and widely distributed in the murine gut microbiota." *ISME J* 12:1127-1141. PMID: 29416123
- Roux S et al. (2015). "Viral dark matter and virus-host interactions resolved from publicly available microbial genomes." *eLife* 4:e08490.
- Touchon M et al. (2016). "Genetic and life-history traits associated with the distribution of prophages in bacteria." *ISME J* 10:2744-2754. PMID: 27015004

## Modular Theory and Phage Evolution

- Dion MB et al. (2020). "Phage diversity, genomics and phylogeny." *Nat Rev Microbiol* 18:125-138.
- Hendrix RW et al. (2000). "Evolutionary relationships among diverse bacteriophages and prophages." *Trends Microbiol* 8:504-508.
- Low SJ et al. (2019). "Evaluation of a concatenated protein phylogeny for classification of tailed double-stranded DNA viruses belonging to the order Caudovirales." *Nat Microbiol* 4:1306-1315.
- Mavrich TN, Hatfull GF (2017). "Bacteriophage evolution differs by host, lifestyle and genome." *Nat Microbiol* 2:17112. PMID: 28692019
- Yutin N et al. (2018). "Discovery of an expansive bacteriophage family that includes the most abundant viruses from the human gut." *Nat Microbiol* 3:38-46.

## Environmental Ecology and Lysogeny

- Camarillo-Guerrero LF et al. (2021). "Massive expansion of human gut bacteriophage diversity." *Cell* 184:1098-1109. PMID: 33606979
- Knowles B et al. (2016). "Lytic to temperate switching of viral communities." *Nature* 531:466-470. PMID: 26982729
- Low SJ et al. (2024). "Prophage-encoded antibiotic resistance genes are enriched in human-impacted environments." *Nat Commun* 15:8178.
- Luque A, Silveira CB (2020). "Quantification of lysogeny caused by phage coinfections in microbial communities from biophysical principles." *mSystems* 5:e00353-20. PMID: 32934113
- Paez-Espino D et al. (2016). "Uncovering Earth's virome." *Nature* 536:425-430. PMID: 27533034

## Defense and Anti-Defense Systems

- Bernheim A, Sorek R (2020). "The pan-immune system of bacteria: antiviral defence as a community resource." *Nat Rev Microbiol* 18:113-119. PMID: 31695182
- Costa AR et al. (2025). "Bacteriophage genomes encode both broad and specific counter-defense repertoires." *Cell Host Microbe*. PMID: 40639337
- Doron S et al. (2018). "Systematic discovery of antiphage defense systems in the microbial pangenome." *Science* 359:eaar4120. PMID: 29371424
- Georjon H, Bernheim A (2023). "The highly diverse antiphage defence systems of bacteria." *Nat Rev Microbiol* 21:686-700.
- Pinilla-Redondo R et al. (2020). "Discovery of multiple anti-CRISPRs highlights anti-defense gene clustering in mobile genetic elements." *Nat Commun* 11:5652. PMID: 33159058
- Pons BJ et al. (2023). "Ecology and evolution of phages encoding anti-CRISPR proteins." *J Mol Biol* 435:167974. PMID: 36690071
- Rahimian M et al. (2026). "Genomic landscape of biosynthetic gene clusters in Iranian extremophiles reveals prolific metabolite potential, prophage associations, and integrated defensive-metabolic islands." *BMC Microbiol*. PMID: 41578173
- Tesson F et al. (2024). "Exploring the diversity of anti-defense systems across prokaryotes, phages, and mobile genetic elements." *Nucleic Acids Res* 53:gkae1171. PMID: 39657785

## Statistical Methods

- Anderson MJ (2001). "A new method for non-parametric multivariate analysis of variance." *Austral Ecol* 26:32-46.
- Borcard D et al. (1992). "Partialling out the spatial component of ecological variation." *Ecology* 73:1045-1055.
- Goberna M, Verdu M (2016). "Predicting microbial traits with phylogenies." *ISME J* 10:959-967.
- Ives AR, Garland T (2010). "Phylogenetic logistic regression for binary dependent variables." *Syst Biol* 59:9-26.

## Prophage Detection Tools

- Camargo AP et al. (2023). "Identification of mobile genetic elements with geNomad." *Nature Biotechnol* 42:1303-1312.
