# References

## Primary Data Source
- Price MN, Arkin AP. (2017). "PaperBLAST: Text Mining Papers for Information about Homologs." *mSystems* 2:e00039-17. PMID: 28845461. https://doi.org/10.1128/msystems.00039-17

## Related PaperBLAST Tools
- Price MN, Arkin AP. (2019). "Curated BLAST for Genomes." *mSystems* 4:e00072-19. PMID: 31164459. https://doi.org/10.1128/mSystems.00072-19
- Price MN, Arkin AP. (2024). "Interactive tools for functional annotation of bacterial genomes." *Database* 2024:baae089. https://doi.org/10.1093/database/baae089

## Research Bias and Understudied Genes
- Stoeger T, Gerlach M, Morimoto RI, Nunes Amaral LA. (2018). "Large-scale investigation of the reasons why potentially important genes are ignored." *PLoS Biol* 16:e2006643. PMID: 30226837. https://doi.org/10.1371/journal.pbio.2006643
- Rocha JJ, Jayaram SA, Stevens TJ, et al. (2023). "Functional unknomics: Systematic screening of conserved genes of unknown function." *PLoS Biol* 21:e3002222. PMID: 37590228. https://doi.org/10.1371/journal.pbio.3002222

## Dark Proteome
- Perdigao N, Heinrich J, Stolte C, et al. (2015). "Unexpected features of the dark proteome." *Proc Natl Acad Sci USA* 112:15898-15903. PMID: 26627242. https://doi.org/10.1073/pnas.1508380112

## KBase Platform
- Arkin AP, Cottingham RW, Henry CS, et al. (2018). "KBase: The United States Department of Energy Systems Biology Knowledgebase." *Nat Biotechnol* 36:566-569. PMID: 29979655. https://doi.org/10.1038/nbt.4163
